src/test/java/com/stripe/functional/LoginLinkTest.java

```diff
@@ -0,0 +1,44 @@
+package com.stripe.functional;
+
+import static org.junit.Assert.assertNotNull;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.exception.StripeException;
+import com.stripe.model.Account;
+import com.stripe.model.LoginLink;
+import com.stripe.model.LoginLinkCollection;
+import com.stripe.net.APIResource;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+public class LoginLinkTest extends BaseStripeTest {
+  public static final String ACCOUNT_ID = "acct_123";
+
+  @Test
+  public void testCreate() throws IOException, StripeException {
+    final Account account = Account.retrieve(ACCOUNT_ID, null);
+
+    // stripe-mock doesn't support this
+    LoginLinkCollection stubbedCollection = new LoginLinkCollection();
+    stubbedCollection.setURL(String.format("/v1/accounts/%s/login_links", account.getId()));
+    account.setLoginLinks(stubbedCollection);
+
+    stubRequest(
+        APIResource.RequestMethod.POST,
+        String.format("/v1/accounts/%s/login_links", account.getId()),
+        null,
+        LoginLink.class,
+        getResourceAsString("/api_fixtures/login_link.json")
+    );
+
+    final LoginLink link = account.getLoginLinks().create();
```

Comment:
I wonder if it'd be cleaner to have an account fixture with the `login_links` collection instead

Reply:
No strong opinion on this -- I don't think it matters too much how the fixtures are generated as long as it doesn't impede readability.